app/src/main/java/com/duckduckgo/app/survey/rmf/TemporaryDefaultSurveyParameters.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2025 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.survey.rmf
+
+import com.duckduckgo.app.browser.senseofprotection.SenseOfProtectionToggles
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.survey.api.SurveyParameterPlugin
+import com.squareup.anvil.annotations.ContributesMultibinding
+import javax.inject.Inject
+
+// Added while senseOfProtectionNewUserExperimentApr25 is active
+@ContributesMultibinding(AppScope::class)
+class SenseOfProtectionCohortSurveyParameterPlugin @Inject constructor(
+    private val senseOfProtectionToggles: SenseOfProtectionToggles,
+) : SurveyParameterPlugin {
+    override val surveyParamKey: String = "senseProtectionCohort"
+
+    override suspend fun evaluate(): String = senseOfProtectionToggles.senseOfProtectionNewUserExperimentApr25().getCohort()?.name.orEmpty()
```

Comment:
@nalcalag shouldn't we checking that experiment is active?

Reply:
If we check if the experiment is active, a cohort will be assigned and we don't want that. We created a new survey for the experiment, so as soon as the experiment is disabled, we should also disable the survey or remove this survey param if they want to keep it.